docs/UserGuide.md

```diff
@@ -617,67 +622,54 @@ In essence, this allows you to find job applicants whose performance rating is a
 Ideally, this feature can then be used to find the best candidates easily and quickly without having to manually look through the list of candidates.
 
 
-
-
 ### Searching job applicants by category: `search`
 
 Finds job applicants whose profiles match the specified categories' keywords. The search categories are: name, status, tag.
 
-Format: `search (n/NAME... / st/STATUS... / t/TAGNAME...)`
-
-#### Searching job applicants by name
-
-Finds job applicants whose names contain the given keywords.
-
-Format: `search n/NAME...`
-
-* Keywords are case-insensitive: `search n/Josh` and `search n/josh` return the same result.
-* Keyword has to be a string that does not contain any non-alpha numeric characters.
-* The order of the keywords does not matter. e.g. `Josh Peck` will match `Peck Josh`.
-* Only full words will be matched e.g. `Jo` will not match `Josh`.
-* Applicants matching at least one keyword will be returned (i.e. `OR` search)
-  e.g. `Josh Peck` will return `Josh Gad`, `Josh Job`.
-
-Examples:
-* `search n/John` returns `john` and `John Doe`
-* `search n/alex david` returns `Alex Yeoh`, `David Li`<br>
-
-#### Searching job applicants by status
-
-Finds job applicants whose status match any of the given keywords.
-
-Format: `search st/STATUS...`
-
-* Keywords can only be from the following list: `Preliminary`, `Interviewed`, `Rejected`, `Offered`
-  e.g. `search st/interviewing` will give an error.
-* Keywords are case-insensitive: `search st/interviewed` and `search st/INTERVIEWED` return the same result.
-
-Example:
-* `search st/interviewed`
-
-#### Searching job applicants by tag
-
-Finds job applicants whose tag(s) match any of the given tag keywords
+Format: `search (n/NAME [MORE NAME] / st/STATUS [MORE STATUS] / t/TAG [MORE TAGS)`
 
-Format: `search t/TAGNAME...`
+Prefix      | Constraints                                                                                                                                               
```

Comment:
Consider adding type (mandatory/optional) column? Can consider doing a merged column filled with mandatory*

Reply:
Regarding this part, technically all fields are optional, but they need at least 1 of the "optional" fields. But I wasn't sure how to word this without making it even more confusing (as flagged by Jun Hiang during our UG consult) so I just got rid of the mandatory/optional column. Any suggestions on how to solve this?

Reply:
I simply added Mandatory* and added a clarifying note below it for delete. You can refer to image below.

<img width="596" alt="image" src="https://github.com/AY2324S1-CS2103T-W09-4/tp/assets/122343294/89467bfd-3918-49d6-9ea7-3d37d9b62400">